integrations/auth0/client.go

```diff
@@ -79,9 +82,42 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 	})
 }
 
-// TODO(INT-113): Implement
-func connTest(_ *integration) sdkintegrations.OptFn {
+// connTest is an optional connection test provided by the integration
+// to AutoKitteh. It is used to verify that the connection is working
+// as expected. The possible results are "OK" and "error".
+func connTest(i *integration) sdkintegrations.OptFn {
 	return sdkintegrations.WithConnectionTest(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
-		return sdktypes.NewStatus(sdktypes.StatusCodeUnspecified, "Not implemented"), nil
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeError, "Init required"), nil
+		}
+
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
```

Comment:
Why do you return error here and below nil error with error status? What's the difference?

Reply:
The distinction is between an error in the server while trying to test the connection, and a connection test failure due to missing/stale auth data.